controllers/argocd/notifications.go

```diff
@@ -65,6 +70,30 @@ func (r *ReconcileArgoCD) reconcileNotificationsController(cr *argoproj.ArgoCD)
 		return err
 	}
 
+	// create clusterrole & clusterrolebinding if cluster-scoped ArgoCD
+	log.Info("reconciling notifications clusterroles")
+	clusterrole, err := r.reconcileNotificationsClusterRole(cr)
+	if err != nil {
+		return err
+	}
+
+	log.Info("reconciling notifications clusterrolebindings")
+	if err := r.reconcileNotificationsClusterRoleBinding(cr, clusterrole, sa); err != nil {
+		return err
+	}
```

Comment:
Can you confirm if we actually need the cluster roles? In the upstream cluster-scoped [manifests](https://github.com/argoproj/argo-cd/blob/master/manifests/install.yaml), I don’t see any cluster role for the `notifications-controller`, so we probably don’t need one.

Reply:
@svghadi That's correct, removed!